python/datasources/vera_incarceration_county.py

```diff
@@ -1,54 +1,49 @@
 import pandas as pd
 from datasources.data_source import DataSource
-from ingestion import gcs_to_bq_util
```

Comment:
Why did you put this vera refactor in this PR? If its a totally separate thing I think it should be split out to a different PR.